src/store/CrosshairStore.js

```diff
@@ -26,43 +26,39 @@ class CrosshairStore {
         return this.context.stx;
     }
 
-
-    @observable top = 0;
-    @observable left = -50000;
-    @observable rows = [];
     @observable state = 2;
-    @observable isArrowLeft = true;
+
     node = null;
     lastBar = {};
     showChange = false;
     showSeries = true;
     showStudies = true;
 
-    hide = () => {
-        this.top = 0;
-        this.left = -50000;
-    };
-
     onContextReady = () => {
         const storedState = this.stx.layout.crosshair;
-        this.state = (typeof storedState !== 'number') ? 2 : storedState;
-        this.setCrosshairState(this.state);
-        this.stx.append('headsUpHR', this.renderCrosshairTooltip);
+        const state = (typeof storedState !== 'number') ? 2 : storedState;
+        this.setCrosshairState(state);
+
+        this.stx.append('headsUpHR',
+            () => this.renderCrosshairTooltip());
```

Comment:
why do we need an extra wrapper for this callback? 🤔

Reply:
Good, point. will fix.